integrations/intercom/src/index.ts

```diff
@@ -19,13 +19,12 @@ const log = console
 const integrationName = 'intercom'
 
 const conversationPartSchema = z.object({
-  type: z.literal('conversation_part'),
   id: z.string(),
   author: z.object({
     id: z.string(),
     type: z.string(),
   }),
-  body: z.string(),
+  body: z.string().nullable(),
 })
 
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** You are modifying the conversation part schema so I also fits a [‏conversation source](https://developers.intercom.com/intercom-api-reference/reference/the-conversation-model#conversation-source). This makes it hard to understand where you are using it. I would either make 2 separate schemas or rename the variables to make it clearer